fiona/ogrext.pyx

```diff
@@ -520,7 +520,7 @@ cdef class Session:
             OGR_L_TestCapability(
                 self.cogr_layer, OLC_STRINGSASUTF8) and
             'utf-8') or (
-            self.get_driver() == "ESRI Shapefile" and
+            "Shapefile" in self.get_driver() and
```

Comment:
It occurred to me that we had a potential bug with respect to the driver name. GDAL allows "Shapefile" or "ESRI Shapefile" and we should handle either.